scripts/update-deps.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash -e
+# Copyright 2022 NVIDIA CORPORATION & AFFILIATES.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o pipefail
+set +e
+usage () {
+	#cat << EOF
+	Usage: $this [-h] [--mofed] [--nvpeer] [--shared-dp] [--sriov-dp] [--cni-plugins] [--multus] [--whereabouts] <version>
+Options:
+  -h         show this help and exit
```

Comment:
Maybe add indentation the same as the others

Reply:
Such indentation is needed to be shown in CLI correctly